jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -469,6 +469,56 @@
               echo "Exiting with code: ${{rc}}"
               exit ${{rc}}
             fi
+      - build-test-kubemark-e2e-gce: # kubernetes-pull-build-test-kubemark-e2e-gce
+          status-context: Kubemark GCE e2e
+          max-total: 12
+          trigger-phrase: 'kubemark\s+gce\s+e2e\s+test'
```

Comment:
It will happen automatically. I would suggest just `@k8s-bot kubemark test this` rather than `kubemark gce e2e test`. I'll never remember that phrase.